components/api_request_helper/api_request_helper.cc

```diff
@@ -8,15 +8,38 @@
 #include <utility>
 
 #include "net/base/load_flags.h"
+#include "services/data_decoder/public/cpp/json_sanitizer.h"
 #include "services/network/public/cpp/resource_request.h"
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 #include "services/network/public/cpp/simple_url_loader.h"
 #include "services/network/public/mojom/url_response_head.mojom.h"
 
 namespace api_request_helper {
 
+namespace {
+
+void OnSanitize(const int http_code,
+                const base::flat_map<std::string, std::string>& headers,
+                APIRequestHelper::ResultCallback result_callback,
+                data_decoder::JsonSanitizer::Result result) {
+  std::string response_body;
+  if (result.error) {
+    VLOG(1) << "Response validation error:" << *result.error;
```

Comment:
this should not be logged if error may contain anything sensitive, like part of the JSON itself

Reply:
@diracdeltas this is a VLOG so it would never be logged without explicitly enabling it. Without the full error I'm not sure the logging would be useful.

Reply:
sgtm